sdk/keyvault/keyvault-keys/tests.yml

```
@@ -7,16 +7,20 @@ extends:
       ServiceDirectory: keyvault
       TimeoutInMinutes: 90
       SupportedClouds: 'Public,UsGov,China'
+      UseFederatedAuth: true
       CloudConfig:
         Public:
           SubscriptionConfiguration: $(sub-config-azure-cloud-test-resources)
           Location: 'eastus2'
+          ServiceConnection: azure-sdk-tests
```

```suggestion
          ServiceConnection: azure-sdk-tests
          SubscriptionConfigurationFilePath: eng/common/TestResources/sub-config/AzurePublicMsft.json
```
I believe this file path is needed when using Federated Auth. As for sovereign clouds, I'm not quite sure what the config is. I saw Java migration PR removed the other clouds in the `SupportedClouds` option. 
cc/ @danieljurek

---

That worked like a charm, thank you so much!